pkg/utils/yaml/loadpolicy.go

```diff
@@ -15,91 +16,104 @@ import (
 )
 
 // GetPolicy extracts policies from YAML bytes
-func GetPolicy(bytes []byte) (policies []kyvernov1.PolicyInterface, validatingAdmissionPolicies []v1alpha1.ValidatingAdmissionPolicy, validatingAdmissionPolicyBindings []v1alpha1.ValidatingAdmissionPolicyBinding, err error) {
+func GetPolicy(bytes []byte) (policies []kyvernov1.PolicyInterface, validatingAdmissionPolicies []v1alpha1.ValidatingAdmissionPolicy, validatingAdmissionPolicyBindings []v1alpha1.ValidatingAdmissionPolicyBinding, policyExceptions []kyvernov2beta1.PolicyException, err error) {
```

Comment:
why are we loading exceptions here ?

Reply:
How can we load it?

Reply:
we can also use [this](https://github.com/kyverno/kyverno/blob/e6d438289ea490b434fa4be4f9638e1c66a59c55/cmd/cli/kubectl-kyverno/exception/load.go#L22) as @eddycharly mentioned in the [issue](https://github.com/kyverno/kyverno/issues/6746#issuecomment-1731088754) conversation but @MariamFahmy98 guide me use GetPolicy function

Reply:
I wasn't aware that we do load exceptions!
In this case, it isn't necessary to load them here as mentioned by Charles.

Reply:
So in this case now i can make changes to use [load](https://github.com/kyverno/kyverno/blob/e6d438289ea490b434fa4be4f9638e1c66a59c55/cmd/cli/kubectl-kyverno/exception/load.go#L22) instead of [GetPolicy](https://github.com/kyverno/kyverno/blob/2f9951ed2657c940680840611a6641d5a3a9f661/pkg/utils/yaml/loadpolicy.go#L18) .
 BTW using GetPolicy seems to be correct and it can already load form stdin,git,http,fs and all kind of policy will be loaded form one function , waiting for your conformations @MariamFahmy98 @eddycharly can i change ?

Reply:
Fixed